### ISSUE TITLE
Error in getting download link through source

### ISSUE THREAD

**Issue by pijozyb** [vikasverma240896/Ravindrababu-GATE-Lectures-Downloader#10]
2018-02-13 18:26:43unknown url type: ''

2018-02-13 18:34:51unknown url type: ''

**Comment by vikasverma240896** [comment]
Are you allowing the script to control your cursor?

**Comment by pijozyb** [comment]
Yes, I am allowing the cursor.
But, in the View Frame Source page, link is not appearing, even when done manually.
Screenshot below.
![ss1](https://user-images.githubusercontent.com/35833897/36268856-a5b00ee8-129d-11e8-951e-34ed30861948.jpg)

**Comment by vikasverma240896** [comment]
What we can do is, start the video once and then check the frame source.

**Comment by vikasverma240896** [comment]
Also, what I've noticed is that, your page source link has attributes like 'title,byline' and stuff which are trying to access the title.
This shouldn't be happening.
Update your browser, clear the cookies and cache and then try running the script.

**Comment by rojansudev** [comment]
I am too facing the same issue.My Chrome is updated and I have cleared the cache and cookies also.

**Comment by vikasverma240896** [comment]
Do you both have a question mark in the framesource url?

**Comment by vikasverma240896** [comment]
Another question, do you see the prompt as below by IDM?
![img](https://user-images.githubusercontent.com/16561198/36339855-6d122988-13f4-11e8-98f1-1b0d1b859b2b.jpeg)

**Comment by rojansudev** [comment]
![screenshot 199](https://user-images.githubusercontent.com/16941075/36339860-7c9c5220-13f4-11e8-9f45-07f4add4453e.png)

**Comment by vikasverma240896** [comment]
@rojansudev do you see the prompt?

**Comment by rojansudev** [comment]
yes, i am able to see the IDM prompt.I am able to download the video using IDM.

**Comment by vikasverma240896** [comment]
That's great, Now I just need to automate that procedure.
You guys need to wait till then.
Also, I think they updated their security after the script came into existence, funny.

**Comment by rojansudev** [comment]
Are you also not able to download now using the script?

**Comment by vikasverma240896** [comment]
I'm, as of now.

By using an ancient chrome version.

**Comment by rojansudev** [comment]
Which version are you using?

**Comment by rojansudev** [comment]
When I manually open the browser and go to the site,the IDM prompt appears,but when selenium opens it,I am not seeing the IDM prompt.

**Comment by vikasverma240896** [comment]
62.0.3202.75

**Comment by pijozyb** [comment]
This seems to be an issue with the latest version of Chrome. How do you prevent Chrome from updating?

**Comment by vikasverma240896** [comment]
Even I feel so. Maybe google will help us with that.

**Comment by rojansudev** [comment]
I downgraded my chrome to the above mentioned one and prevented it from updating.Still it is not working.

**Comment by vikasverma240896** [comment]
There's a minuscule thing we're unable to figure out. Because I updated my browser and still the script works like a charm.

**Comment by rojansudev** [comment]
you are using which python version of python 3 and selenium ,pyautogui and requests module?

**Comment by vikasverma240896** [comment]
Python 3.5.0
pyautogui 0.9.36
selenium 3.8.0
requests in pre-installed I suppose.

But I think, the versions hardly matter. It's something else I believe.

**Comment by rojansudev** [comment]
I am using python 3.6
